apps/test/levelTests.js

```diff
@@ -11,32 +11,105 @@
 
 var path = require('path');
 var assert = require('chai').assert;
-var wrench = require('wrench');
-
-var child_process = require('child_process');
+var $ = require('jquery');
 
 var testUtils = require('./util/testUtils');
 testUtils.setupLocales();
 
-getTestCollections('./test/solutions').forEach(function (path) {
-  runTestCollection(path);
+var wrappedEventListener = require('./util/wrappedEventListener');
+
+// One day this might be the sort of thing we share with initApp.js
+function loadSource(src) {
+  var deferred = new $.Deferred();
+  document.body.appendChild($('<script>', { src: src }).on('load', function () {
+    deferred.resolve();
+  })[0]);
+  return deferred;
+}
```

Comment:
agreed we'd ideally like to code-share this functionality. If a good package doesn't already exist on npm for this kind of behavior, this could be a good case for testing out code-sharing across our internal projects using a tiny npm package that we load as a dependency from multiple places. this looks fine for now though